src/wp_ecx_kmgmt.c

```diff
@@ -1674,7 +1677,7 @@ static int wp_ed448_import_private(const byte* in, word32 inLen,
 /** Ed448 data and wolfSSL functions. */
 static const wp_EcxData ed448Data = {
     WP_KEY_TYPE_ED448,
-    448,
+    456,
```

Comment:
This looks like it is ED448_KEY_SIZE * 8.
Use defines for this and above in ED25519.